src/components/CountdownTimer.tsx

```
@@ -0,0 +1,44 @@
+    }, 1000);
+  }, []);
+
+  return (
+    <div className="flex gap-6">
+      {[
+        { label: "Days", value: timeLeft.days },
+        { label: "Hours", value: timeLeft.hours },
+        { label: "Minutes", value: timeLeft.minutes },
+        { label: "Seconds", value: timeLeft.seconds },
+      ].map((item) => (
+        <div className="flex flex-col items-center">
```

Biome (2.3.13)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)